### PR TITLE
Added CI Workflow & Attempt at creating test package

### PR DESCRIPTION
**Pull request by nardineshak**

### CODE REVIEW

File: .github/workflows/pullrequestCI.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        dotnet-version: ['3.0', '3.1.x', '5.0.x' ]
```
Comment (marcomagdy) [inline .github/workflows/pullrequestCI.yml]: We're using .net 7.0, so we should change these.

File: Psaltos.Tests/TestClassAttribute.cs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+namespace Psaltos.Tests
+{
+    internal class TestClassAttribute : Attribute
+    {
+    }
+}
```
Comment (marcomagdy) [inline Psaltos.Tests/TestClassAttribute.cs]: I don't think we need this class or file. You can remove it.

File: Psaltos.Tests/TestMethodAttribute.cs [hunk 3]
```diff
@@ -0,0 +1,6 @@
+namespace Psaltos.Tests
+{
+    internal class TestMethodAttribute : Attribute
```
Comment (marcomagdy) [inline Psaltos.Tests/TestMethodAttribute.cs]: Same here, I don't think we need this file/class.

File: psaltos.csproj [hunk 4]
```diff
@@ -4,11 +4,21 @@
...
     <Nullable>enable</Nullable>
     <ImplicitUsings>enable</ImplicitUsings>
+    <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
+    <GenerateAssemblyTitleAttribute>false</GenerateAssemblyTitleAttribute>
+    <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
+    <GenerateAssemblyInformationalVersionAttribute>false</GenerateAssemblyInformationalVersionAttribute>
+    <GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
+    <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
+    <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
+    <GenerateTargetFrameworkAttribute>false</GenerateTargetFrameworkAttribute>
```
Comment (marcomagdy) [inline psaltos.csproj:22]: Nice. I remember googling for these :)

**Review by marcomagdy — DISMISSED** [review]
Thank you for doing the work @nardineshak.
This looks great, I left a couple of comments.

**Comment by AthanIsaac** [comment]
LGTM

**Review by AthanIsaac — APPROVED** [review]

**Pull request merged**